db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+fake_gender = ["Male", "Female"]
+
+(0..5).each do |i|
+  author = Author.create(first_name: Faker::Name.first_name,
+                         last_name: Faker::Name.last_name,
+                         birth_country: Faker::Address.country,
+                         gender: fake_gender[i % 2],
+                         website: Faker::Internet.url("goodreads.com"),
+                         twitter_username: "@" + Faker::Lorem.word,
+                         author_biography: Faker::Lorem.sentence)
+  (0..7).each do |j|
```

Comment:
Unused block argument - `j`. You can omit the argument if you don't care about it.